server/plugin/command_test.go

```diff
@@ -63,21 +75,154 @@ func TestExecuteCommand(t *testing.T) {
 			isConnected:      true,
 			ephemeralMessage: constants.UserDisconnected,
 		},
+		{
+			description:      "ExecuteCommand: boards command when user is not connected",
+			commandArgs:      &model.CommandArgs{Command: "/azuredevops boards wrong [title] [description]"},
+			ephemeralMessage: fmt.Sprintf(constants.ConnectAccountFirst, fmt.Sprintf(constants.ConnectAccount, p.GetPluginURLPath(), constants.PathOAuthConnect)),
+		},
+		{
+			description:      "ExecuteCommand: invalid boards command",
+			isConnected:      true,
+			commandArgs:      &model.CommandArgs{Command: "/azuredevops boards wrong [title] [description]"},
+			ephemeralMessage: constants.InvalidCommand,
+		},
 		{
 			description:      "ExecuteCommand: boards create command",
+			isConnected:      true,
 			commandArgs:      &model.CommandArgs{Command: "/azuredevops boards create [title] [description]"},
 			ephemeralMessage: fmt.Sprintf(constants.ConnectAccountFirst, fmt.Sprintf(constants.ConnectAccount, p.GetPluginURLPath(), constants.PathOAuthConnect)),
 		},
+		{
+			description: "ExecuteCommand: boards add subscription command",
+			isConnected: true,
+			commandArgs: &model.CommandArgs{Command: "/azuredevops boards subscription add"},
+		},
+		{
+			description:     "ExecuteCommand: boards delete subscription command",
+			isConnected:     true,
+			commandArgs:     &model.CommandArgs{Command: "/azuredevops boards subscription delete mockSubscriptionID"},
+			isDeleteCommand: true,
+			serviceType:     "boards",
+		},
+		{
+			description:                  "ExecuteCommand: failed to delete subscription from store",
+			isConnected:                  true,
+			commandArgs:                  &model.CommandArgs{Command: "/azuredevops boards subscription delete mockSubscriptionID"},
+			isDeleteCommand:              true,
+			serviceType:                  "boards",
+			deleteSubscriptionStoreError: errors.New("failed to delete subscription from store"),
+		},
+		{
+			description:             "ExecuteCommand: failed to get all subscriptions while deleting a subscription",
+			isConnected:             true,
+			commandArgs:             &model.CommandArgs{Command: "/azuredevops boards subscription delete mockSubscriptionID"},
+			isListCommand:           true,
+			serviceType:             "boards",
+			getAllSubscriptionError: errors.New("failed to get all subscriptions while deleting a subscription"),
+			ephemeralMessage:        constants.GenericErrorMessage,
+		},
+		{
+			description:      "ExecuteCommand: boards list subscription command",
+			isConnected:      true,
+			commandArgs:      &model.CommandArgs{Command: "/azuredevops boards subscription list me"},
+			isListCommand:    true,
+			ephemeralMessage: "mockSubscriptionList",
+		},
+		{
+			description:             "ExecuteCommand: failed to get all subscriptions",
```

Comment:
Check this everywhere
```suggestion
			description:             "ExecuteCommand: failed to get all the subscriptions",
```